### PR TITLE
Fix Wine plugin "Override DLL"

### PR DESCRIPTION
**Pull request by plata**
see #1151

### CODE REVIEW

**Comment by plata** [comment]
@Zemogiter please try.

File: Engines/Wine/Plugins/override DLL/script.js [hunk 1]
```diff
@@ -3,23 +3,21 @@ const Regedit = include("engines.wine.plugins.regedit");
 module.default = class OverrideDLL {
     constructor(wine) {
         this.wine = wine;
-        this.modes = {};
+        this.modes = new Map();
```
Comment (madoar) [inline Engines/Wine/Plugins/override DLL/script.js]: Indeed a `Map` is a better choice here.

File: Engines/Wine/Plugins/override DLL/script.js [hunk 2]
```diff
@@ -3,23 +3,21 @@ const Regedit = include("engines.wine.plugins.regedit");
...
+        this.modes.set(mode, libraries);
 
         return this;
     }
 
     go() {
         let regeditFileContent = `REGEDIT4\n\n[HKEY_CURRENT_USER\\Software\\Wine\\DllOverrides]\n`;
 
-        Object.entries(this.modes)
-            .map(([mode, libraries]) => libraries.map(library => [library, mode]))
```
Comment (madoar) [inline Engines/Wine/Plugins/override DLL/script.js]: Just as a sidenot, I think the following change would be enough to fix the issue but I agree that your code is more readable:

```javascript
.flatMap(([mode, libraries]) => libraries.map(library => [library, mode]))
```

**Comment by Zemogiter** [comment]
@plata thanks now it works

**Pull request merged**